README.md

```diff
@@ -40,54 +40,126 @@ git clone https://github.com/fortran-lang/stdlib
 cd stdlib
 ```
 
+
 ### Requirements
 
-The preprocessor ```fypp``` (https://github.com/aradi/fypp) is needed because metaprogramming is used.
-It can be installed using the command line installer ```pip```.
+To build the Fortran standard library you need
+
+- a Fortran 2008 compliant compiler, or better, a Fortran 2018 compliant compiler
+  (GCC Fortran and Intel Fortran compilers are known to work for stdlib)
+- CMake version 3.14 or newer (alternatively Make can be used)
+- a build backend for CMake, like Make or Ninja (the latter is recommended on Windows)
+- the [fypp](https://github.com/aradi/fypp) preprocessor (used as meta-programming tool)
+
+If your system package manager does not provide the required build tools, all build dependencies can be installed with the Python command line installer ``pip``:
+
 ```sh
-pip install fypp
+pip install --user fypp cmake ninja
 ```
 
+Alternatively, you can install the build tools from the conda-forge channel with the conda package manager:
+
+```sh
+conda config --add channels conda-forge
+conda create -n stdlib-tools fypp cmake ninja
+conda activate stdlib-tools
+```
+
+You can install conda using the [miniforge installer](https://github.com/conda-forge/miniforge/releases).
+Also, you can install a Fortran compiler from conda-forge by installing the ``fortran-compiler`` package, which installs GFortran.
+
+
+### Supported Compilers
+
+The following combinations are tested on the default branch of stdlib:
+
+Name | Version | Platform | Architecture
+--- | --- | --- | ---
+GCC Fortran | 7, 8, 9, 10 | Ubuntu 18.04 | x86_64
+GCC Fortran | 7, 8, 9, 10 | MacOS Catalina 10.15 | x86_64
```

Comment:
GFortran 7 have issues to compile stdlib. See, e.g., #58 , #61, #171, #296.
Therefore I suggest to remove it from this lis ant to add it to the next one.

```suggestion
GCC Fortran | 8, 9, 10 | Ubuntu 18.04 | x86_64
GCC Fortran | 8, 9, 10 | MacOS Catalina 10.15 | x86_64
```

Reply:
It is currently tested in the CI, therefore I assumed it should work.

Reply:
You are right. It is still tested in the CI. It should be probably removed from it.

Reply:
GCC 7.5 is working nicely, the breakage described in the linked issues happens with previous versions of the GCC 7 series.